render-utils.js

```diff
@@ -0,0 +1,24 @@
+export function renderPost(post) {
+    const li = document.createElement('li');
+
+    // const a = document.createElement('a');
+    // a.href = `/post/?id=${post.id}`;
```

Comment:
why is this commented out? that's why you missed 3 points for 'Posts linked to Post Detail Page'